Backend/app/rider-platform/rider-app/Main/spec/Storage/FrfsTicket.yaml

```diff
@@ -488,6 +503,10 @@ FRFSConfig:
     freeTicketInterval: "9999"
     maxFreeTicketCashback: "0"
     busStationTtl: "1800"
+    straightLineDistance: "4"
+    radius: "1"
+    straightLineDistance: "4"
```

Comment:
Repeated twice. Can be reverted.
Also Default values keep as below:
straightLineDistance: 5000
radius: 3000

Reply:
Yes, sorry bhaiya, I forgot, that these are of Meters type